### DISCUSSION TITLE
'Uncaught TypeError: Cannot read properties of undefined (reading 'prototype')' when using bluelinky package in a vue project

### DISCUSSION THREAD

**Discussion by neelpatil09** [Hacksore/bluelinky#214] [Q&A] [answered]
Hello, I am using vue.js to make a simple button on a webpage that locks my Hyundai car through the bluelinky package. I have a bluelink account with Hyundai. Before the app is even rendered, I run into the following error:

`index.js:54 Uncaught TypeError: Cannot read properties of undefined (reading 'prototype')
    at node_modules/duplexer3/index.js (index.js:54:55)
    at __require2 (chunk-ICLR7M5L.js?v=dc3311d4:33:50)
    at node_modules/got/source/as-stream.js (as-stream.js:3:19)
    at __require2 (chunk-ICLR7M5L.js?v=dc3311d4:33:50)
    at node_modules/got/source/create.js (create.js:3:18)
    at __require2 (chunk-ICLR7M5L.js?v=dc3311d4:33:50)
    at node_modules/got/source/index.js (index.js:3:16)
    at __require2 (chunk-ICLR7M5L.js?v=dc3311d4:33:50)
    at node_modules/bluelinky/dist/index.js (index.js:2:90)
    at __require2 (chunk-ICLR7M5L.js?v=dc3311d4:33:50)
node_modules/duplexer3/index.js @ index.js:54
__require2 @ chunk-ICLR7M5L.js?v=dc3311d4:33
node_modules/got/source/as-stream.js @ as-stream.js:3
__require2 @ chunk-ICLR7M5L.js?v=dc3311d4:33
node_modules/got/source/create.js @ create.js:3
__require2 @ chunk-ICLR7M5L.js?v=dc3311d4:33
node_modules/got/source/index.js @ index.js:3
__require2 @ chunk-ICLR7M5L.js?v=dc3311d4:33
node_modules/bluelinky/dist/index.js @ index.js:2
__require2 @ chunk-ICLR7M5L.js?v=dc3311d4:33
(anonymous) @ dep:bluelinky:1`

 My npm and node versions are up to date and I have experimented using Duplexer2 as well. I am unsure why the prototype object is undefined as my app is very basic at the moment and just makes a bluelinky object and attempts to connect to my car first. I would appreciate any help on this as there is little documentation for errors like this on the internet. Thank you!

**Comment by Hacksore** [comment, marked as answer]
@neelpatil09 Are you trying to use `bluelinky` in your vue (client-side) app? Currently, bluelinky is for node environments right now.

**Reply by neelpatil09** [reply]
I understand, it works now on my end. Thank you for the quick response!
